pnpm-lock.yaml

```diff
@@ -1572,6 +1575,28 @@ importers:
         specifier: 5.5.2
         version: 5.5.2
 
+  packages/postgrest:
```

Comment:
maybe call it db?

Reply:
If we will start move onto then yes. Not a pig deal as it's a   "private": true package